reporter/src/main/kotlin/utils/SpdxDocumentModelMapper.kt

```diff
@@ -211,6 +215,25 @@ private fun ScanResult.toSpdxPackageVerificationCode(): SpdxPackageVerificationC
         packageVerificationCodeValue = summary.packageVerificationCode.toLowerCase()
     )
 
+private fun SpdxDocument.addExtractedLicenseInfo(licenseTextProvider: LicenseTextProvider): SpdxDocument {
+    val nonSpdxLicenses = packages.flatMapTo(mutableSetOf()) {
+        SpdxExpression.parse(it.licenseConcluded).licenses() + SpdxExpression.parse(it.licenseDeclared).licenses()
+    }.filter {
+        !SpdxConstants.isNotPresent(it) && SpdxLicense.forId(it) == null && SpdxLicenseException.forId(it) == null
+    }
+
+    val extractedLicenseInfo = nonSpdxLicenses.sorted().mapNotNull { license ->
```

Comment:
Hint: If you're sorting `nonSpdxLicenses` here, you could also map to a `sortedSetOf()` above instead.

Reply:
I'd like to leave it as is.